hc/accounts/tests/test_check_token.py

```diff
@@ -22,7 +22,44 @@ def test_it_redirects(self):
         self.assertEqual(self.profile.token, "")
 
     ### Login and test it redirects already logged in
+    def test_it_redirects_already_logged_in_provided_we_do_a_request_with_the_token_again(self):
+        """Tests if user is already logged in. Visiting the token link again
+        should return him to his dashboard
+        :returns: link should be valid as long as user is logged in: True
+        """
+        login = self.client.post("/accounts/check_token/alice/secret-token/")
+        self.assertRedirects(login, "/checks/")
+
+        already_logged = self.client.get("/") #  check to see if user will be redirected to dashboard
+        self.assertRedirects(already_logged, "/checks/")
+
+        login_again = self.client.get("/accounts/check_token/alice/secret-token/") # the same token should work again
+        self.assertRedirects(login_again, "/checks/")
 
     ### Login with a bad token and check that it redirects
+    def test_when_a_user_login_with_bad_token(self):
+        """
+        Test if user will be logged in with provided that he gave a bad token
```

Comment:
@sir3n-sn kindly pay attention to details - the test name and description are not well expressed

Reply:
Thanks i will work on them